dashboard/config/scripts/v2_u1l12_student_lesson_introduction.external

```diff
@@ -5,32 +5,30 @@ css 'unplugged'
 skip_dialog 'true'
 
 markdown <<MARKDOWN
-## Unit 1: Lesson 12 - The Need for DNS
+# Unit 1: Lesson 12 - The Need for DNS
 
-# Background
+## Background
 
 It would be impossible for packets to be routed across the internet without IP addresses. The problem with this system is that, while computers are good at referring to other computers by numbers, humans are not. The Domain Name System (DNS) solves this problem so that we can identify a webpage by its name, even when the IP addresses change.
 
-# Vocabulary
+## Vocabulary
 
-<strong>DNS</strong>: an abbreviation for Domain Name System, the Internet's system for converting alphabetic names into numeric IP addresses. 
+* **DNS:** an abbreviation for Domain Name System, the Internet's system for converting alphabetic names into numeric IP addresses.
```

Comment:
Nope, prefer markdown `**DNS**` over HTML `<strong>DNS</strong>`.  Also moved the colon inside the tag.